packages/manager/modules/config/src/environment/environment.constants.js

```diff
@@ -1,5 +1,6 @@
 export const ALLOWED_REGIONS = ['EU', 'CA', 'US'];
 export const DEFAULT_REGION = 'EU';
+export const CANADA_REGION = 'CA';
```

Comment:
The constant `CANADA_REGION` is exported but never used in `@ovh-ux/manager-config` package, and it's never exported in the dist.

We don't have `EUROPE_REGION` or `UNITED_STATES_REGION`. If we add them, we should at least export them in the dist (as `REGIONS:{EU, CA, US}` , and refactor every place in our code where `EU` / `CA` / `US` strings are used.